pkg/services/rendering/rendering.go

```diff
@@ -429,6 +429,18 @@ func (rs *RenderingService) getGrafanaCallbackURL(path string) string {
 		return fmt.Sprintf("%s%s&render=1", rs.Cfg.RendererCallbackUrl, path)
 	}
 
+	if rs.Cfg.RendererCallbackUrl != "" {
+		_, err := url.Parse(rs.Cfg.RendererCallbackUrl)
+		if err != nil {
+			rs.log.Warn("Image renderer callback url is not valid. " +
```

Comment:
I think this log line should be added here: https://github.com/grafana/grafana/blob/ec34041652a3e1fb752047486ed8cc063194a494/pkg/services/rendering/rendering.go#L87 and we shouldn't parse again the URL as we don't need it here. 
We could then combine this check with the if above as it returns the same thing.